firedrake/adjoint_utils/checkpointing.py

```diff
@@ -73,21 +73,22 @@ def enable_disk_checkpointing(dirname=None, comm=COMM_WORLD, cleanup=True):
 
 
 def disk_checkpointing():
-    """Return true if disk checkpointing of the adjoint tape is active."""
-    return _stop_disk_checkpointing <= 0
+    """Return whether disk checkpointing is enabled."""
+    global _enable_disk_checkpoint
```

Comment:
```suggestion
```
I think that, since you aren't modifying the value, this line isn't needed.

Reply:
Yes. That is right.